housingbayarea/bloom
# 3138/multiselect question x translation issue

## Issue Overview

This PR addresses [#issue](https://github.com/bloom-housing/bloom/issues/3138)


## Description
This should resolve the creation of a new application where the application is translated into any language besides english.
The problem was that when an application was submitted from the public site while that application was translated into not english the data in the program/preference column would save in the not english language.

This resolves the actual saving of new applications, a follow on pr will come with a migration to clean the data

## How Can This Be Tested/Reviewed?
go to create a new application on the public site
select a language besides english

continue on with the application creation process
1) make sure the application form has translations for the program/preference sections of the application process
2) make sure the review step of the application has the program/preference sections translated
3) make sure you can submit the application
4) on the partner side verify the data that was submitted is present
5) verify the csv export of that application has no column for a program/preference that is translated

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
